### ISSUE TITLE
2 of 4 sliders are not compliant with WCAG SC 2.5.2

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#2446]
The 2 are changed only at Up-Event:

* [Vertical Temperature Slider Example](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-temperature.html)
* [Media Seek Slider Example](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-seek.html)

The 2 are not changed on the down event, but they are changed when the pointing device is moved, so it is not possible to cancel it
* [Color Viewer Slider Example](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-color-viewer.html)
* [Rating Slider Example](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-rating.html)

**Comment by charmarkk** [comment]
On 8/30/22, the APG discussed this issue. 

<details>
<summary> IRC Log: </summary>

[13:54] <MarkMcCarthy> Matt_King: if this is correct, sounds like there's a bug in our sliders
[13:55] <MarkMcCarthy> Matt_King: i'd have thought they were all using the same JS, curious
[13:55] <MarkMcCarthy> Matt_King: I trust JAWS-test, so there's probably a bug somewhere.
[13:56] <MarkMcCarthy> jamesn: but a native HTML range responds to a down event too
[13:56] <MarkMcCarthy> Matt_King: are we using one?
[13:56] <MarkMcCarthy> jamesn: if even the native stuff doesn't comply here...
[13:56] <MarkMcCarthy> Matt_King: then that's a browser bug
[13:57] <MarkMcCarthy> Matt_King: regardless - if our sliders aren't compliant, i'd like them to be.
[13:57] <MarkMcCarthy> Matt_King: we'd need some mousers to test and look at this
[13:58] <MarkMcCarthy> jamesn: reading through 2.5.2, I don't think this applies. it can be undone by resetting the slider's position
[13:58] <MarkMcCarthy> jamesn: Abort or Undo: Completion of the function is on the up-event, and a mechanism is available to abort the function before completion or to **undo the function after completion**;
[13:58] <MarkMcCarthy> jamesn: is moving the slider a function? not sure
[13:58] <MarkMcCarthy> Siri: I thought that'd only apply to buttons that adjust a slider, not one with thumbs
[13:59] <MarkMcCarthy> jamesn: right - i don't think this applies here
[14:00] <MarkMcCarthy> jamesn: if you were dragging a slider, and you move the cursor off and the slider resets, that'd be really *weird*
[14:00] <MarkMcCarthy> Matt_King: and would hinder users with jitters etc.
[14:00] <MarkMcCarthy> Matt_King: either way, interesting that there are 2 that work that way and 2 that don't
[14:01] <MarkMcCarthy> Matt_King: sounds like the ones that don't comply are just less user friendly?
[14:01] <MarkMcCarthy> jamesn: maybe JAWS-test means if you click *on the slider itself* and move away... it changes on.mouse*UP* one 2 of them, and on.mouse*DOWN* on the other two. if that's the case, that's a reasonable change
[14:02] <MarkMcCarthy> jamesn: in that case, things should be consistent.
[14:02] <MarkMcCarthy> Siri: i didn't even think about using a slider that way, i'm glad you pointed it out
[14:03] <MarkMcCarthy> jamesn: still may not be a failure, since you can reset it, even if you don't know the initial state, and that could be an improvement
[14:03] <MarkMcCarthy> jamesn: there could be many ways to build in an undo, too
[14:03] <MarkMcCarthy> Matt_King: so the problem seems to be NOT with dragging the pointer, it's with a single click on the rail
[14:03] <MarkMcCarthy> jamesn: i think so
[14:05] <MarkMcCarthy> jamesn: we need the down event though, you can't get fine adjustments without it
[14:06] <MarkMcCarthy> curt: i agree - it'd be nonsensical to move it only on the up event
[14:06] <MarkMcCarthy> jamesn: the exception in the SC is that the value/slider can be reset
</details>

**Comment by mcking65** [comment]
Based on meeting discussion, changing issue title. No WCAG issue here, but there is inconsistency in the implementation that should be corrected. Thumb should move on down event when rail is clicked. This is not WCAG failure; it is usability feature and consistent with native HTML range implementations.
